Dockerfile

```diff
@@ -1,15 +1,37 @@
-FROM golang:alpine
+ARG GO_VERSION=1.17
+ARG ALPINE_VERSION=3.15
+
+# Build stage
+FROM golang:${GO_VERSION}-alpine${ALPINE_VERSION} as build
 
 WORKDIR /app
 
-COPY go.mod .
-COPY go.sum .
+COPY go.mod go.sum .
+
 RUN go mod download
 
-COPY . .
+COPY cmd cmd
+
+COPY server server
+
+RUN CGO_ENABLED=0 go build \
+    -ldflags="-extldflags=-static" \
+    -o /devportal cmd/devportal/main.go
+
+
+# Final stage
+FROM gcr.io/distroless/static as final
```

Comment:
I've thought about this more and I don't see any benefits to multi-stage builds for us since the Docker configuration is only meant for development. They don't save disk space because the build image is downloaded anyway. If there's another reason to use them, let me know

Reply:
When you build locally, it indeed does not save disk space.
But what you download is the final image, you don't download any previously used layer.

Reply:
Yes, but we're not publishing images. The Dockerfile is only for local development

Reply:
Okay. I thought that maybe the image could serve both purposes. In that case I don't think multi-stages build is needed.